### ISSUE TITLE
Does not support Cucumber version > 0.7.0

### ISSUE THREAD

**Issue by gkushang** [simondean/parallel-cucumber-js#16]
@simondean we ran into error for the cucumber latest version 0.9.2. It looks like Cucumber is updated and Parallel-Cucumber fails to run tests. Below is the error. It would be great if module is updated with cucumber latest versions.

```
/Users/gkushang/Documents/workspace/authnodeweb/node_modules/cucumber/lib/cucumber/cli/configuration.js:8
  if (args.length > 0) {
          ^
TypeError: Cannot read property 'length' of undefined
    at Function.Configuration (/Users/gkushang/Documents/workspace/authnodeweb/node_modules/cucumber/lib/cucumber/cli/configuration.js:8:11)
    at Object.self._task (/Users/gkushang/Documents/workspace/authnodeweb/node_modules/parallel-cucumber/lib/parallel_cucumber_worker/cli/main.js:134:43)
    at process.<anonymous> (/Users/gkushang/Documents/workspace/authnodeweb/node_modules/parallel-cucumber/lib/parallel_cucumber_worker/cli/main.js:16:12)
    at process.emit (events.js:110:17)
    at handleMessage (child_process.js:324:10)
    at Pipe.channel.onread (child_process.js:352:11)

```

Latest Cucumber Configuration function takes Two Arguments instead of one

```
function Configuration(options, args) {
  var Cucumber = require('../../cucumber');
  var path = require('path');
  var fs = require('fs');
  var _ = require('lodash');

  var unexpandedFeaturePaths = ['features'];
  if (args.length > 0) {

```

**Comment by gkushang** [comment]
Submitted a PR for this issue: https://github.com/simondean/parallel-cucumber-js/pull/17

**Comment by nidhimj22** [comment]
+1

**Comment by gkushang** [comment]
@nidhimj22 you can use https://www.npmjs.com/package/cucumber-parallel module which supports the latest cucumber version for the Parallel executions

**Comment by simondean** [comment]
@nidhimj22 pull request #19 is going to add support for cucumber-js 0.10.x

**Comment by simondean** [comment]
Closed by #c113a57

**Comment by nidhimj22** [comment]
Thanks!
